Implement Protocol Changes for Tree.RemoveStyle Feature Support

raararaara opened the pull request:
<!--  Thanks for sending a pull request! -->

#### What this PR does / why we need it?


#### Any background context you want to provide?


#### What are the relevant tickets?
<!--
*Automatically closes linked issue when PR is merged.
Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
-->
Related #745, https://github.com/yorkie-team/yorkie/pull/748

### Checklist
- [ ] Added relevant tests or not required
- [ ] Didn't break anything

hackerwins reviewed (commented):
@raararaara Is there no need to add related test codes?

raararaara commented:
> @raararaara Is there no need to add related test codes?
> 
> Please also add a brief explanation.
> 
> > What this PR does / why we need it?
> > Any background context you want to provide?
> > What are the relevant tickets?
> > Related #745, [yorkie-team/yorkie#748](https://github.com/yorkie-team/yorkie/pull/748)

I have added the relavent test code and added a brief explanation.

hackerwins reviewed (approved):
LGTM.
